Dockerfile

```diff
@@ -25,8 +25,11 @@ RUN groupadd -r node \
 ENV USER node
 USER node
 
+ARG PORT=9090
+ENV HDD_SUPERVISOR_PORT ${PORT}
```

Comment:
Do you need changes to cico_setup.sh for this to be picked up even during as part of CI/CD pipelines for image build?

Reply:
oh, I missed that.

Reply:
Hey, changes made!!!!

I've append the changes to infra_test script too.